src/schematools/permissions/db.py

```diff
@@ -690,6 +693,10 @@ def _get_sequence_name(session: Session, table_name: str, column: str) -> str |
         return value
 
 
-def scope_to_role(scope: str) -> str:
+def scope_to_role(scope: str | Scope) -> str:
     """Return rolename for the postgres database."""
-    return f"scope_{scope.lower().replace('/', '_')}"
+    if isinstance(scope, str):
+        return f"scope_{scope.lower().replace('/', '_')}"
+    # productiePackage has the pattern "EM4W-DATA-schemascope-p-scope_some_auth_level".
+    # We need the last part including the "scope_" prefix.
+    return scope.productiePackage.split("-")[-1]
```

Comment:
We weten niet wat de naamgeving van de groepen wordt die de teams zelf gaan aanmaken. 

Ik denk dat dit obv het id veld van de scope moet zijn:

```suggestion
    return f"scope_{scope.id.lower().replace('/', '_')}"
```